.circleci/config.yml

```diff
@@ -291,14 +291,22 @@ jobs:
       - setup_remote_docker:
           docker_layer_caching: true
       - build-temp-image
-      - snyk/scan:
-          token-variable: SNYK_TOKEN
-          docker-image-name: vcd:scan
-          target-file: ./docker/Dockerfile
-          organization: "legal-aid-agency"
-          severity-threshold: "critical"
-          fail-on-issues: true
-          no-output-timeout: 20m
+      - run:
```

Comment:
Why the change from the orb to a hand-written command? If it's just following a snippet elsewhere, I think it's worth looking at what the actual change is and seeing if we can just incorporate that into what we already have.

Is the point of it to get the `policy-path`?

Reply:
it was an experiment, which didn't work. 
`fail-on-issues: false` seems a temporary viable option to me.

Reply:
I reset that to true, and removed the Cypress lib.